src/main/resources/db/migration/V73__Add_last_known_addresses_table.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE last_known_address (
+          id UUID PRIMARY KEY,
+          recall_id UUID NOT NULL REFERENCES recall (id),
+          line1 TEXT NOT NULL,
+          line2 TEXT NULL,
+          town TEXT NOT NULL,
+          postcode TEXT NULL,
+          source TEXT NOT NULL,
+          index INTEGER NOT NULL,
+          created_by_user_id UUID references user_details(id),
+          created_date_time TIMESTAMPTZ NOT NULL,
```

Comment:
Just looking to tighten this to:
          created_by_user_id UUID NOT NULL references user_details(id),

Reply:
Added